integrated_channels/blackboard/models.py

```diff
@@ -25,20 +25,20 @@ class BlackboardEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfigur
     client_id = models.CharField(
         max_length=255,
         null=True,
-        verbose_name="API Client ID",
+        verbose_name="API Client ID or Blackboard Application Key",
```

Comment:
yup I guess it could confuse a bit, would it be okay if I cleaned this up in the next coming pr? only because changing this would need to regen migration (instead of creating a new one I would need to clean up and regen I think?)